app/models/item.rb

```diff
@@ -84,11 +84,11 @@ def duplicate_revisions
   def persist_revision
     if content_type == 'Note'
       revision = Revision.new
-      revision.content = content
-      revision.content_type = content_type
-      revision.enc_item_key = enc_item_key
-      revision.items_key_id = items_key_id
-      revision.auth_hash = auth_hash
+      revision.content = content_before_last_save
```

Comment:
What would the result of this be for new items which have never been saved before?

Reply:
revision with empty state

Reply:
the spec reflects that

Reply:
So the client UI would always show an empty revision for each item?

Reply:
🤔 we can add a condition that previous content has to be not empty

Reply:
Right, that or just checking if uuid_before_last_save is null.

Reply:
Fixed :)

Reply:
Fixed